apps/condo/domains/user/utils/limits/resetters/AuthGuardResetter.js

```diff
@@ -1,11 +1,11 @@
-const { PHONE_TYPE, IPv4_TYPE } = require('@condo/domains/user/constants/identifiers')
+const { IPv4_TYPE, PHONE_TYPE, EMAIL_TYPE } = require('@condo/domains/user/constants/identifiers')
 const { AUTH_COUNTER_LIMIT_TYPE } = require('@condo/domains/user/constants/limits')
 
 const { RedisGuardResetter } = require('./RedisGuardResetter')
 
 class AuthGuardResetter extends RedisGuardResetter {
     constructor () {
-        super(AUTH_COUNTER_LIMIT_TYPE, [PHONE_TYPE, IPv4_TYPE])
+        super(AUTH_COUNTER_LIMIT_TYPE, [IPv4_TYPE, PHONE_TYPE, EMAIL_TYPE])
```

Comment:
```suggestion
        super(AUTH_COUNTER_LIMIT_TYPE, [IPv4_TYPE])
```

You're using it only with IPs, so why do you need reset email / phone?

Reply:
It is for the future when we add other abilities for logging in.

Reply:
YAGNI